docs/root/faq/configuration/timeouts.rst

```diff
@@ -28,7 +28,7 @@ Connection timeouts apply to the entire HTTP connection and all streams the conn
   <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.common_http_protocol_options>`
   field in the HTTP connection manager configuration. To modify the idle timeout for upstream
   connections use the
-  :ref:`common_http_protocol_options <envoy_v3_api_field_extensions.upstreams.http.v3.HttpProtocolOptions.common_http_protocol_options>` field in the Cluster's :ref:`extension_protocol_options<envoy_v3_api_field_config.cluster.v3.Cluster.typed_extension_protocol_options>`, keyed by `envoy.extensions.upstreams.http.v3.HttpProtocolOptions`
+  :ref:`common_http_protocol_options <envoy_v3_api_field_extensions.upstreams.http.v3.HttpProtocolOptions.common_http_protocol_options>` field in the Cluster's :ref:`extension_protocol_options<envoy_v3_api_field_config.cluster.v3.Cluster.typed_extension_protocol_options>`, keyed by :ref:`HttpProtocolOptions <envoy_v3_api_msg_extensions.transport_sockets.tls.v3.UpstreamTlsContext>`
```

Comment:
Can you double check that the link to `HttpProtocolOptions` is the correct one?

Reply:
fixed